src/backend/distributed/executor/executor.c

```diff
@@ -562,7 +562,7 @@ FinishDistributedExecution(DistributedExecution *execution)
 
 		if (connection->connectionState == MULTI_CONNECTION_CONNECTING)
 		{
-			ShutdownConnection(connection);
```

Comment:
ShutdownConnection is used in a few other places. Are those usages also incorrect?